packages/esm-patient-chart-app/src/visit-header/visit-header.component.tsx

```diff
@@ -59,15 +59,10 @@ const PatientInfo: React.FC<PatientInfoProps> = ({ patient }) => {
   const priority = queueEntry?.priority ?? '';
 
   const getServiceString = useCallback(() => {
-    switch (queueEntry?.status?.toLowerCase()) {
-      case 'waiting':
-        return `Waiting for ${queueEntry.service}`;
-      case 'in service':
-        return `Attending ${queueEntry.service}`;
-      case 'finished service':
-        return `Finished ${queueEntry.service}`;
-      default:
-        return '';
+    if (queueEntry?.status) {
+      return `${queueEntry.status} - ${queueEntry.service}`;
```

Comment:
Is the `-` here necessary? We kind of want to end up in a state where it's possible to display "Waiting for xxx", "Attending xxx", etc.

Reply:
My other thought here is that it would be a good idea to run this through the translation system:

```ts
return `${t(queueEntry.status)} ${t(queueEntry.service)}`
```

This allows implementations to customize the text if they don't want to customize the concept.

Reply:
It should also be null-safe I realise.  eg. we should prepare for the possibility that queueEntry.service could be null.

Reply:
Thanks @ibacher and @mseaton ! I have added the translate function, and also checked that queueEntry.service is not null.
@ibacher , the reason I added the '-' (hyphen) here is to be consistent with the other change we made to the service-queue: 
https://github.com/openmrs/openmrs-esm-patient-management/blob/main/packages/esm-service-queues-app/src/helpers/functions.ts#L20

If there is a clearer pattern for displaying statuses, I am happy to update the code in both places so the queue statuses will be displayed in a consistent way. Please let me know. Thanks!

Reply:
It's an interesting point about the hyphen, @ibacher , but as @cioan  pointed out we are making this consistent with another change we already made, and although I can see the desire to support the display strings of status + service to represent a phrase, I don't think we we to mandate this, if that makes any sense?